src/containers/ContainerEventsForAdmin.js

```diff
@@ -0,0 +1,8 @@
+import { connect } from "react-redux";
+import EventsForAdmin from "../components/event/EventsForAdmin/EventsForAdmin";
+
+const mapStateToProps = ({ user }) => {
+  return { id: user.id };
```

Comment:
Does it make sense to create a separate container to get user.id?

Reply:
Yes, it makes sense to create for a connection with redux store. 
In the future it will be easier to refactor the component into a functional one.